### DISCUSSION TITLE
Works great

### DISCUSSION THREAD

**Discussion by anthnyajp** [JohnStrunk/wahoo-results#588] [Show and tell]
We're a summer swim club team and we purchased a 70" smart TV before this season to use as our scoreboard after testing this software at the end of last season and it was a huge hit with families. It also reduced the interruptions from swimmers and families asking about times or current score. 

As a work around, as suggest by John Strunk, we're utilizing the title field to update the scoring as we're always only doing dual meets. You can see from the attached photos. Note, the one photo that has a time but no name is because we combined our boys and girls 15-18 events so that was expected. We use Meet Maestro, so I don't believe there's any workaround for this besides manually updating the exported files. 

![image](https://github.com/JohnStrunk/wahoo-results/assets/11774820/656a17cd-d3e4-422e-a9fc-948490d22f9e)

![image](https://github.com/JohnStrunk/wahoo-results/assets/11774820/e58b5a62-c8d5-41c9-a822-262e924fcce3)

![image](https://github.com/JohnStrunk/wahoo-results/assets/11774820/a04baf34-a727-4b73-8ef3-d02df7ddafa4)

**Comment by JohnStrunk** [comment]
That looks awesome! Thanks for the pics, @anthnyajp.
I like the custom colors and logo :shark:
